README.md

```diff
@@ -31,7 +31,7 @@ Below are instructions to build on different OS's. You can also use the followin
 ```powershell
 PS> git submodule update --init
 PS> .\submodules\vcpkg\bootstrap-vcpkg.bat
-PS> .\submodules\vcpkg\vcpkg.exe install cpprestsdk:x64-windows
+PS> .\submodules\vcpkg\vcpkg.exe install cpprestsdk:x64-windows msgpack:x64-windows
```

Comment:
Remove, if you want you can add this to the `-DUSE_MSGPACK` section in the table above.

Reply:
Visual Studio shows errors regarding msg-pack then it try to parse cmakelists on folder-project opening. Maybe it is not an issue.